src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -149,90 +149,61 @@ them, and how to use them to configure your environment.
-protocol. The workaround is to use a PAMR router, or the HTTP protocol.
-
-For that reason, the HTTP protocol is forced for communications going
-from the RM to the EC2 nodes. On the other way, from EC2 to the RM,
-using HTTP communications is also the simplest, also RMISSH is possible
-but requires manual configuration for deploying RSA keys. To configure
-your RM with HTTP, all you need to specify is a configuration file with
-the following properties, for both Resource Manager *and* Scheduler.
-
-``` {.xml}
-<prop key="proactive.net.nolocal" value="true"/>
-<prop key="proactive.communication.protocol" value="http" />
-<prop key="proactive.http.port" value="PORT" />
-<prop key="proactive.net.noprivate" value="true" />
-<prop key="proactive.useIPaddress" value="true" />
-```
-
-Note that you need to specify a different port for both configurations.
-Sample configurations can be found in */samples/ec2*.
+    
+    -   *operatingSystem:* The operating system running in the AWS instance.
+    
+    -   *downloadCommand:* The command to download the Proactive node.jar. This command is executed in all the newly created AWS instances at cloud init time. 
+    	The URL needs to be accessible from the AWS cloud.
+    	
+    -   *additionalProperties:* Additional java command properties to be added when connecting the node.jar to the resource manager (e.g. \"-Dpropertyname=propertyvalue\").
+     
+    -   *ram:* The desired RAM amount expressed in Mega Bytes for each AWS instance that needs to be created.
+    
+    -   *cpu:* The desired CPU amount amount expressed in units for each AWS instance that needs to be created.
+    
++
+WARNING: If the combination between RAM and CPU does not match any existing AWS instance type, then the closest to the specified parameters will be selected.
++
+    
 
 Using this configuration, you can start a Resource Manager and a
 Scheduler using the */bin/proactive-server* script. An EC2 NodeSource can
-now be added using the command line interface of the Resource Manager:
+now be added using the *Create Node Source* panel in the Resource Manager or the command line interface:
 
-    $ PROACTIVE_HOME/bin/proactive-client --createns ec2 -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.EC2Infrastructure ./config/rm/deployment/ec2.properties rmURL RMCredentialsPath nodeHttpPort
+    $ PROACTIVE_HOME/bin/proactive-client --createns ec2 -infrastructure org.ow2.proactive.resourcemanager.nodesource.infrastructure.AWSEC2Infrastructure aws_key aws_secret_key rmDomain connectorIaasURL image numberOfInstances numberOfNodesPerInstance operatingSystem downloadCommand additionalProperties ram cpu 
 
-You will need to specify:
 
--   **configurationFile** - EC2 configuration file such as
-    */config/rm/deployment/ec2.properties*, which you should have filled
-    with your AWS credentials beforehand.
+Once the Resource Manager is up and running with its EC2 NodeSource, new
```

Comment:
I think this sentence is obvious and is out of the scope of the explanations for creating an EC2 infrastructure.